README.md

```diff
@@ -86,14 +88,15 @@ cp config.pri.default config.pri
 # Edit config.pri
 ```
 
- * open QField.pro with QtCreator (installed during OSGeo4A installation)
- * hit build
+ * Open CMakeList.txt with QtCreator (installed during OSGeo4A installation).
+ * Hit build.
+
+If you make your own QGIS build, use the following variables: `QGIS_ANALYSIS_LIBRARY`, `QGIS_CORE_LIBRARY`, `QGIS_INCLUDE_DIR`, and `QGIS_PLUGIN_DIR`.
```

Comment:
Why don't we put `QGIS_PREFIX_PATH` as an easier way to setup?

Reply:
https://github.com/opengisch/QField/pull/1257/files#diff-04c6e90faac2675aa89e2176d2eec7d8R100 I think it's `QGIS_INSTALL_PATH`

Reply:
https://github.com/opengisch/QField/blob/master/cmake/FindQGIS.cmake#L109 I mean this one, it's 4 more times in the file.

Reply:
I've confirmed the instructions above work on my machines, I'm not confident suggesting something else I haven't tried.

Should we merge this and have you update the readme again when you can confirm it works for you? Or have you already confirmed it works post qmake death?

Reply:
I use the one that I suggested above since I started writing for QField, including post qmake age. But I am also fine to merge it as it is and make another PR later, so I am responsible for the change.